lib/config/error-codes/index.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { InternalError: KuzzleInternalError } = require('kuzzle-common-objects').errors;
+const api = require('./api');
+
+/** Check the format of the error-codes config files
+ *  @param {object} - error config file domain
+ */
+
+function checkErrorCodes(errorCodes) {
+
+  for (const domain of Object.keys(errorCodes)) {
+    const domainName = errorCodes[domain];
+    if (!(domainName.hasOwnProperty('subdomains')) || typeof domainName['subdomains'] !== 'object' 
+      || !(domainName.hasOwnProperty('code')) || typeof domainName['code'] !== 'number' 
+      || !(domainName['code'] > 0 && domainName['code'] < 10)) {
```

Comment:
this is an hexadecimal code, so its range is 00->FF
```suggestion
      || !(domainName['code'] > 0 && domainName['code'] < 10)) {
```

Reply:
Why hexa ? Since json does not support it, why we can't keep decimals?

Reply:
it's only the range, just convert it to base10 to get the maximum

we devised the code to be easily transportable in a 16 bits buffer, which is pretty handy with low-level languages such as C

Reply:
oh ok thanks